packages/ember-metal/lib/keys.js

```diff
@@ -7,7 +7,7 @@
   @for Ember
   @param {Object} obj
   @return {Array} Array containing keys of obj
-  @private
+  @public
```

Comment:
Lets make it `[DOC release]` and just deal with `Ember.compare`.  We can remove the file in a separate `[CLEANUP beta]` PR if you don't mind...